whist/core/cards/deck.py

```diff
@@ -9,8 +9,6 @@
 class Deck:
     """An unordered collection of cards"""
 
-    __cards: set[Card]
```

Comment:
why you remove?
i thought you liked type hints :P

Reply:
Yes we, but don't want to add another pylint ignore.
Can we add them below?

Reply:
We actually move __cards to the init with self.__cards and than do the type hinting